`/webclient/standalone` example builds OK but running the separate client as in the `README.md` fails

[//]: # "Please remove these comments"
[//]: # "Provide a general summary of the issue in the Title above"

## Environment Details
* Helidon Version: 3.x
* Helidon SE or Helidon MP
* JDK version:
* OS:
* Docker version (if applicable):

----------

## Problem Description
The `/webclient/standalone` app builds and tests successfully, but it includes both a server and a client. When you start the server and then run the client per the instructions in the `README.md` file the client reports a failure in the redirection step.

## Steps to reproduce
In the `helidon-examples` repo clone:
```
cd examples/webclient/standalone
mvn clean package
java -jar target/helidon-examples-webclient-standalone.jar
```
Note the port number displayed (XXX).
In another terminal window:
```
java -cp "target/classes:target/libs/*" io.helidon.examples.webclient.standalone.ClientMain XXX
```
You get output with a few successes and then a failure at the redirection step:
```
Exception in thread "main" java.util.concurrent.CompletionException: java.lang.IllegalStateException: Follow redirection failed!
	at java.base/java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413)
	at java.base/java.util.concurrent.CompletableFuture.join(CompletableFuture.java:2118)
	at io.helidon.common.reactive.Awaitable.await(Awaitable.java:54)
	at io.helidon.examples.webclient.standalone.ClientMain.main(ClientMain.java:103)
Caused by: java.lang.IllegalStateException: Follow redirection failed!
	at io.helidon.examples.webclient.standalone.ClientMain.lambda$followRedirects$7(ClientMain.java:132)
```

--- a/examples/webclient/standalone/src/main/java/io/helidon/examples/webclient/standalone/ServerMain.java
+++ b/examples/webclient/standalone/src/main/java/io/helidon/examples/webclient/standalone/ServerMain.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, 2023 Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2024 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -41,7 +41,7 @@ public final class ServerMain {
      * @param args starting arguments
      */
     public static void main(String[] args) {
-        startServer();
+        startServer().await();
     }
 
     /**

